common/common.mk

```diff
@@ -23,6 +23,11 @@ GOVERSION?=$(shell curl -fsSL "https://raw.githubusercontent.com/containerd/cont
 endif
 
 GOLANG_IMAGE=golang:$(GOVERSION)
+ifeq ($(OS),Windows_NT)
+       GOLANG_IMAGE=docker.io/library/golang:$(GOVERSION)
+else
+       GOLANG_IMAGE=docker.io/library/golang:$(GOVERSION)-buster
```

Comment:
This is a workaround for golang:1.12.x images  where the manifest list is missing s390x. With golang:1.13.x or newer this if block can ve removed again.

Reply:
can we make tis conditional on Go 1.12.x as well then? We are in process of switching as 1.12 is out of support.